test/integration/spec/docker/reconnection.js

```diff
@@ -98,10 +96,8 @@ async function setup(setupOptions) {
     const token = getToken(identity);
 
     if (region) {
-      options.iceServers = await waitFor(
-        getRegionalizedIceServers(token, region, options),
-        `${sid}: get TURN servers regionalized to ${region}`);
       options.iceTransportPolicy = 'relay';
+      options.region = region;
```

Comment:
Didn't we already do this for fixing the stable network tests? What did we do to make the stable network tests pass?

Reply:
This didn't initially work because we didn't have the proper IPs that we were connecting to. After getting those updated addresses, it started working. The stable network tests were another issue, I believe.